public/components/patientSignUp.js

```diff
@@ -101,7 +101,7 @@ if(formSubmit()) {
 //     }
 //     if (i > 1) {  
 //         alert('Is Working');
-//         window.location = "https://hackathonv4.glitch.me/FirebaseRTC/public/patWaitingRoom.html";      
+//         window.location = "https://hackathonv4.glitch.me/public/templates/patWaitingRoom.html";      
```

Comment:
If this comment is going to stay, I think this line needs to be the same. The glitch project hasn't actually changed.

Either delete the comment or revert the change on this line so that at least the comment is logically correct.

Reply:
Done